resources/public/api/conf/2.0/application.yaml

```diff
@@ -0,0 +1,1797 @@
+    # Overview
+    Use the CTC Traders API to send departure and arrival movement to the New Computerised Transit System (NCTS)
+    and to get messages sent from customs offices of departure and destination.
+servers:
+  - url: https://test-api.service.hmrc.gov.uk
+    description: Sandbox
+  - url: https://api.service.hmrc.gov.uk
+    description: Production
+    variables: {}
+paths:
+  /customs/transits/movements/departures:
+    post:
+      tags:
+      - Departures
+      summary: Send a declaration data message
+      description: >
+        Send a transit declaration to the Customs Office of Departure to notify it that you wish to carry out a transit operation.
+
+        This request is also called an ‘Declaration Data’ E_DEC_DAT (IE015) message.
+
+        A successful response will contain a URI which you can use to get the message status. A successful response can also include an optional box ID, which you can use to match this movement to any messages received from the Push Pull Notifications API.
+         <br/><br/>
+        **Note:** If an Economic Operators Registration and Identification (EORI) number has not been assigned to you, an HTTP 403 Forbidden error is returned.
+      operationId: postDeclarationDataMessage
+      parameters:
+        - $ref: '#/components/parameters/acceptHeader'
+        - $ref: '#/components/parameters/contentXmlJsonHeader'
+      responses:
+        '202':
+          description: A successful departure declaration response
+          content:
+            application/json:
+              schema:
+                allOf:
+                - $ref: '#/components/schemas/CustomsTransitsMovementsDeparturesResponse'
+                - description: JSON payload
+                  example:
+                    _links:
+                      self:
+                        href: /customs/transits/movements/departures/62fb8f33d233578e
+                      messages:
+                        href: /customs/transits/movements/departures/62fb8f33d233578e/messages
+                    boxId: b54c2424-955b-496c-bbce-dfb9f57cd797
+              example:
+                _links:
+                  self:
+                    href: /customs/transits/movements/departures/62fb8f33d233578e
+                  messages:
+                    href: /customs/transits/movements/departures/62fb8f33d233578e/messages
+                boxId: b54c2424-955b-496c-bbce-dfb9f57cd797
+        '413':
+          description: Request Entity Too Large
+          content:
+            application/json:
+              schema:
+                allOf:
+                 - $ref: '#/components/schemas/RequestEntityTooLarge'
+                 - example:
+                    code: REQUEST_ENTITY_TOO_LARGE
+                    message: Your message size must be less than 500000 bytes
+              example:
+                code: REQUEST_ENTITY_TOO_LARGE
+                message: Your message size must be less than 500000 bytes
+        '400':

```

Comment:
Should this be listed above `413` error? I am thinking about ease of maintenance.

Reply:
I think this is in correct order. As first we get 413 then 400? Please suggest if you still not ok.